src/box.ts

```diff
@@ -275,7 +276,8 @@ export const stbl = (track: Track) => box('stbl', null, [
 	stss(track),
 	stsc(track),
 	stsz(track),
-	stco(track)
+	stco(track),
+	ctts(track)
```

Comment:
Is there a way we can avoid emitting this box if all composition time offsets are 0? Then the box tree is a little more lean.